addons/binding/org.openhab.binding.solaredge/README.md

```diff
@@ -6,6 +6,11 @@ Only read access is supported.
 ## Supported Things
 
 This binding provides only one thing type: "generic" which is the inverter itself. As the name suggests it is generic which means it applies to all available inverters. Create one inverter thing per physical inverter installation available in your home(s). Additional stuff like batteries is automatically supported.
+Inverters which have a meter attached allow more detailed measuring. Either a SolarEdge modbus meter or a S0 meter (export or consumption meter) can be used. While on the one hand the S0 meter is the cheaper solution the solaredge meter on the other hand can be used as combined import+export meter and therefore allows even more detailed measurements.
```

Comment:
Please use one line per sentence here.
The website build will care of the representation during the build process.